vault/resource_pki_secret_backend_role.go

```diff
@@ -106,6 +106,13 @@ func pkiSecretBackendRoleResource() *schema.Resource {
 				Description: "Flag to allow any name",
 				Default:     false,
 			},
+			"allow_wildcard_certificates": {
+				Type:        schema.TypeBool,
+				Required:    false,
+				Optional:    true,
+				Description: "Flag to allow wildcard certificates",
+				Default:     true,
```

Comment:
Can this defaulted to `false`? I think this is preferred because we don't want to default to allowing more permissive certificate generation. This means that people already using this resource will need to explicitly set `allow_wildcard_certificates` to `true` if they were depending on this behavior (bc the API defaults to true)

Reply:
That would be a breaking change requiring a major version bump of the provider.

And IMHO it would be better to follow the defaults of the Vault API, even though I agree that `false` would have been more secure default there.

But the Vault team makes the call, of course.

Reply:
Let's keep the default the same as the API